mob/mob.h

```diff
@@ -11,6 +11,9 @@
 
 enum item_type {Weapon, Potion, Creature};
 
+#define NUM_MONSTER_TYPES 4
```

Comment:
I would follow the convention I use in chemestry.h where the count const is the last item in the enum + 1. That makes it easier to add things to the enum later.

Reply:
Dang. Everyone on the net suggested this but I thought it was too silly. Wrong again. : )

Reply:
Oh. That's not what they suggest. They suggest making the last element something like "NUM_ELEMENTS" and not doing a '+1'.

That actually seems better to me because nobody will accidentally add an element after 'ash' and then weird things happen.  WDYT?

Reply:
I'm putting it into the next push on this PR so you can see it.